README.rst

```diff
@@ -38,6 +38,8 @@ to ask us anything there. We have a very welcoming and helpful community.
 Download
 --------
 
+Installation methods for Windows, MacOS and Linux are available here :- "https://docs.sympy.org/latest/documentation-style-guide.html#installation"
+
```

Comment:
Isn't this the instructions for installing a development environment for building the documentation? I don't think that is what we want to point end users to.